clients/src/test/java/org/apache/kafka/common/security/oauthbearer/internals/OAuthBearerClientInitialResponseTest.java

```diff
@@ -40,16 +77,6 @@ public void testAuthorizationId() throws Exception {
         assertEquals("myuser", response.authorizationId());
     }
 
-    @Test
```

Comment:
Why delete this test?  Can you just call `response.extensions().map().get()` instead of `response.propertyValue()`?